rust_snuba/src/processors/replays.rs

```diff
@@ -11,51 +11,59 @@ pub fn process_message(
     metadata: KafkaMessageMetadata,
 ) -> anyhow::Result<InsertBatch> {
     let payload_bytes = payload.payload().context("Expected payload")?;
+    let replay_row = deserialize_message(payload_bytes, metadata.partition, metadata.offset)?;
+    InsertBatch::from_rows([replay_row])
```

Comment:
I don’t think this is what you want, rather:
```suggestion
    let replay_rows = deserialize_message(payload_bytes, metadata.partition, metadata.offset)?;
    InsertBatch::from_rows(replay_rows)
```

Reply:
Yes you're right it did not need to be double wrapped.  Totally missed that.